packages/sanity/src/core/i18n/bundles/studio.ts

```diff
@@ -68,6 +68,310 @@ export const studioLocaleStrings = {
+
+  /** Label for placeholder when searching specific document types */
+  'navbar.search.action.search-for-doc-type': `Search for {{documentTypes}}`,
+
+  /** Label for searching all documents */
+  'navbar.search.action.search-all-docs': `Search all documents`,
+
+  /** Label for header (accessibility) when it's full screen and you want to close the search */
+  'navbar.search.action.close-search': 'Close search',
+
+  /** Label for header (accessibility) when it's full screen and you want to toggle filters */
+  'navbar.search.action.toggle-filters': 'Toggle filters',
+
+  /** Label for instructions on how to use the search when no recent searches are available */
+  'navbar.search.instructions': 'Use <0></0> to refine your search',
+
+  /** Label for truncating document types: showing all types */
+  'navbar.search.all-types-label': 'All types',
+
+  /** Label for truncating document types: showing the remaining count of document types */
+  'navbar.search.remaining-document-types': '+{{remainingCount}} more',
+
+  /** --- Configuration Issues --- */
+
+  /** label for when when there are configuration issues */
+  'navbar.configuration.error.found-configuration-issues-status': 'Found configuration issues',
+
+  /** Error title for when there are configuration issues */
+  'navbar.configuration.error.configuration-issues-title': 'Configuration issues',
+
+  /** Error description for when there are configuration issues, explaining that the checks are only performed during development */
+  'navbar.configuration.error.configuration-issues-description':
+    'Configuration checks are only performed during development and will not be visible in production builds',
+
+  /** Warning label that tells the user how many warnings were found */
+  'navbar.configuration.found-number-schema-warning': `Found {{count}} schema warnings`,
+
+  /** Label for displaying the schema error and warnings for the studio configurations */
+  'navbar.configuration.type-label': 'type',
+
+  /** Prompt to view documentation about the schema problems */
+  'navbar.configuration.action.view-documentation': 'View documentation',
+
+  /** --- Help & Resources Menu --- */
+
+  /** Title for help and resources menus */
+  'navbar.helpResources.title': 'Help and resources',
+
+  /** Information for what studio version the current studio is running */
+  'navbar.helpResources.studio-version': `Sanity Studio version {{studioVersion}}`,
+
+  /** Information for what the latest sanity version is */
+  'navbar.helpResources.latest-sanity-version': `Latest version is {{latestVersion}}`,
+
+  /** Label for "join our community" call to action */
+  'navbar.helpResources.action.join-our-community': `Join our community`,
+
+  /** Label for "help and support" call to action */
+  'navbar.helpResources.action.help-and-support': `Help and support`,
+
+  /** Label for "contact sales" call to action */
+  'navbar.helpResources.action.contact-sales': `Contact sales`,
```

Comment:
**note:** these are titles for _fallback_ links in the event the help + resources endpoint isn't able to be fetched. Help and resources links and copy are _not_ localized in the admin studio side, so we'll probably want to address this at some point!

**suggestion:** considering updating comment and/or the names to reflect the fallback nature of these

Reply:
Updating the comment to reflect this 👍